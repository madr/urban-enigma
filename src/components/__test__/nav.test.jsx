import React from "react";
import renderer from "react-test-renderer";
import Nav from "../nav";

it("renders correctly", () => {
  const tree = renderer.create(<Nav goto={() => {}} />).toJSON();

  expect(tree).toMatchSnapshot();
});
