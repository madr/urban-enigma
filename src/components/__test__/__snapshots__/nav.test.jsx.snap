// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  id="nav"
>
  <button
    onClick={[Function]}
    title="Historik"
  >
    ðŸ“†
  </button>
  <button
    onClick={[Function]}
    title="Logga pass"
  >
    âž•
  </button>
  <button
    onClick={[Function]}
    title="Kalkylator fÃ¶r 1rm"
  >
    1RM
  </button>
</nav>
`;
